cli/tre/commands/login.py

```diff
@@ -118,6 +119,7 @@ def login_device_code(base_url: str, client_id: str, aad_tenant_id: str, api_sco
         'aad-tenant-id': aad_tenant_id,
         'api-scope': api_scope,
         'verify': verify,
+        'cloud': cloud,
```

Comment:
So the new param is only relevant for login?
Have you tested that a logged-in cli of the previous version is still functional after this new param has been added without doing a new login?

Reply:
it's only relevant for login and it will fail if we have an already logged-in cli session.
However, I have found a better way to get the cloud environment - we can get it from az CLI using this package:
https://pypi.org/project/azure-cli-core/